src/test/db/createUser.js

```diff
@@ -46,9 +46,17 @@ const createUser = async () => {
   up.isFirstTimelog = true;
   up.actualEmail = '';
   up.isVisible = true;
-  up._id = '65cf6c3706d8ac105827bb2e';
 
-  await up.save();
+  /* 
```

Comment:
This is fine. This update will conflict with my PR, but I will my PR accordingly. to make sure it doesn't break your tests. Alternatively, I will create another file just to hardoce the _id for testing purposes.

Reply:
I see, I thought it would just be more reusable to have it create new _id's in the event that we need to create multiple users to test functionality.

Reply:
That makes sense. I will make sure this file is untouched, and will create a new one for my own unit tests.